Formula/grpc.rb

```diff
@@ -1,8 +1,8 @@
 class Grpc < Formula
```

Comment:
Btw, it looks like that grpc formula is no longer depending on google-protobuf  but on protobuf.  Not sure if that's a good thing or not.